test/ably/restpush_test.py

```diff
@@ -310,3 +308,23 @@ def test_admin_channel_subscriptions_save(self):
         subscription = PushChannelSubscription(channel, device_id='notregistered')
         with pytest.raises(AblyException):
             save(subscription)
+
+    # RSH1c4
+    def test_admin_channel_subscriptions_remove(self):
+        save = self.ably.push.admin.channel_subscriptions.save
+        remove = self.ably.push.admin.channel_subscriptions.remove
+
+        channel = 'canpublish:testremove'
+
+        # Subscribe device
+        device = self.get_device()
+        subscription = save(PushChannelSubscription(channel, device_id=device.id))
+        assert remove(subscription).status_code == 204
+
+        # Subscribe client
+        client_id = self.get_client_id()
+        subscription = save(PushChannelSubscription(channel, client_id=client_id))
+        assert remove(subscription).status_code == 204
+
+        # Remove again, it doesn't fail
+        assert remove(subscription).status_code == 204
```

Comment:
I think you need to do a get / list after removing to make sure it's actually gone. Just checking you get `204` confirms the API request succeeded, but for example could be a call to another and incorrect API.